android-exoplayer/src/main/java/com/brentvatne/exoplayer/ReactExoplayerViewManager.java

```diff
@@ -314,6 +316,26 @@ public void setControls(final ReactExoplayerView videoView, final boolean contro
         videoView.setControls(controls);
     }
 
+    @ReactProp(name = PROP_ENABLE_MEDIA_SESSION, defaultBoolean = false)
+    public void setEnabledMediaSession(final ReactExoplayerView videoView, final boolean enableMediaSession) {
```

Comment:
I'm not sure, but wanted to double check just in case. Is there a spelling error here?

```suggestion
    public void setEnableMediaSession(final ReactExoplayerView videoView, final boolean enableMediaSession) {
```